{{ cookiecutter and 'tmp_repo' }}/.gitignore

```diff
@@ -1,18 +1,44 @@
-# SPDX-FileCopyrightText: 2017 Scott Shawcroft, written for Adafruit Industries
+# SPDX-FileCopyrightText: 2022 Kattni Rembor, written for Adafruit Industries
 #
-# SPDX-License-Identifier: Unlicense
+# SPDX-License-Identifier: MIT
 
+# Do not include files and directories created by your personal work environment, such as the IDE
+# you use, except for those already listed here. Pull requests including changes to this file will
+# not be accepted.
+
+# This .gitignore file contains rules for files generated by working with CircuitPython libraries,
+# including building Sphinx, testing with pip, and creating a virual environment, as well as the
+# MacOS and IDE-specific files generated by using MacOS in general, or the PyCharm or VSCode IDEs.
+
+# If you find that there are files being generated on your machine that should not be included in
+# your git commit, you should create a .gitignore_global file on your computer to include the
+# files created by your personal setup.
+# First, create a file called .gitignore_global somewhere convenient for you, and add rules for
+# the files you want to exclude from git commits.
+# Second, configure Git to use the exclude file for all Git repositories by running the
+# following via commandline, replacing "path_to_your" with the actual path to your newly created
+# .gitignore_global file:
+# git config --global core.excludesfile path_to_your/.gitignore_global
+
+# CircuitPython-specific files
 *.mpy
-.idea
+
+# Python-specific files
 __pycache__
-_build
 *.pyc
+
+# Sphinx build-specific files
+_build
+
+# This file results from running `pip -e install .` in a local repository
+*.egg-info
```

Comment:
This is stretching my knowledge here.

I think this might need to be `**/*.egg-info`, the leading `**` I think matches to any directory name, the inner `*` matches to any filename prefix ending in `.egg-info`. I don't know how many of the circuit python libraries might have multiple interior directories, but I think these wildcards handle any library directory structure

Reply:
Interesting! That's what we had and I changed it. I'll swap it back. Thanks!